firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -1308,6 +1306,8 @@ class BrowserViewController: UIViewController,
         } else {
             adjustBottomSearchBarForKeyboard()
         }
+
+        super.updateViewConstraints()
```

Comment:
curious, why move the update here?

Reply:
It's marked as an important step specified in documentation when overriding this method.